<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1006</width>
    <height>754</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">*{
	background: rgb(18, 10, 30);
	border-radius: 10px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>35</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="topMargin">
           <number>6</number>
          </property>
          <property name="rightMargin">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>6</number>
          </property>
          <item>
           <widget class="QFrame" name="frame_4">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label">
               <property name="styleSheet">
                <string notr="true">*{
	color: white;
	font-family: Noto Sans;
	font-size: 14px;
}</string>
               </property>
               <property name="text">
                <string>Avocado Price Predictor - Lol</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_5">
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="reduceButton">
               <property name="maximumSize">
                <size>
                 <width>15</width>
                 <height>15</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">*{
	background-color: rgb(84, 255, 117);
	border-radius: 7px;
	border: none;
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="maximizeButton">
               <property name="maximumSize">
                <size>
                 <width>15</width>
                 <height>15</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">*{
	background-color: rgb(255, 152, 55);
	border-radius: 7px;
	border: none;
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="exitButton">
               <property name="maximumSize">
                <size>
                 <width>15</width>
                 <height>15</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">*{
	background-color: rgb(255, 55, 55);
	border-radius: 7px;
	border: none;
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QScrollArea" name="scrollArea">
            <property name="styleSheet">
             <string notr="true">QScrollBar:vertical {
	width: 15px;
	border-radius: 7px;
	border: none;
	margin: 15px 0 15px 0;
	background: rgb(29, 16, 48);
}

QScrollBar:handle:vertical {
	background: rgb(57, 37, 98);
	border-radius: 7px;
}

QScrollBar::sub-line:vertical {
	background: rgb(57, 37, 98);
	height: 15px;
	border-top-left-radius: 7px;
	border-top-right-radius: 7px;
	subcontrol-position: top;
	subcontrol-origin: margin;
}

QScrollBar::add-line:vertical {
	background: rgb(29, 16, 48);
	height: 15px;
	border-top-left-radius: 7px;
	border-top-right-radius: 7px;
	subcontrol-position: top;
	subcontrol-origin: margin;
}

QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical {
	background: none;
}

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
	background: none;
}</string>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>-208</y>
               <width>955</width>
               <height>888</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <property name="spacing">
               <number>10</number>
              </property>
              <item>
               <widget class="QFrame" name="frame_10">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>100</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">*{
	background-color: rgb(29, 16, 48);
}</string>
                </property>
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_4">
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="styleSheet">
                    <string notr="true">*{
	color: white;
	font-family: Noto Sans;
	font-size: 16px;
}</string>
                   </property>
                   <property name="text">
                    <string>Saisissez le nombre total d'avocats vendus.</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="totalVolume">
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>35</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">*{
	color: rgb(197, 197, 197);
	background: rgba(0, 0, 0, 0.0);
	border: 2px solid;
	border-radius: 1px;
	border-bottom-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:1, stop:0 rgba(255, 121, 215, 255), stop:1 rgba(145, 94, 255, 255));
	border-top-color: rgba(0, 0, 0, 0.0);
	border-right-color: rgba(0, 0, 0, 0.0);
	border-left-color: rgba(0, 0, 0, 0.0);
}</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                   </property>
                   <property name="placeholderText">
                    <string>Nombre total d'avocats vendu</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QFrame" name="frame_6">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>100</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">*{
	background-color: rgb(29, 16, 48);
}</string>
                </property>
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_5">
                 <item>
                  <widget class="QLabel" name="label_3">
                   <property name="styleSheet">
                    <string notr="true">*{
	color: white;
	font-family: Noto Sans;
	font-size: 16px;
}</string>
                   </property>
                   <property name="text">
                    <string>Saisissez le nombre total d'avocats avec PLU 4096 vendus</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="plu_4097">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>35</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">*{
	color: rgb(197, 197, 197);
	background: rgba(0, 0, 0, 0.0);
	border: 2px solid;
	border-radius: 1px;
	border-bottom-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:1, stop:0 rgba(255, 121, 215, 255), stop:1 rgba(145, 94, 255, 255));
	border-top-color: rgba(0, 0, 0, 0.0);
	border-right-color: rgba(0, 0, 0, 0.0);
	border-left-color: rgba(0, 0, 0, 0.0);
}</string>
                   </property>
                   <property name="placeholderText">
                    <string>Nombre total d'avocats avec PLU 4046 vendus </string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QFrame" name="frame_7">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>100</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">*{
	background-color: rgb(29, 16, 48);
}</string>
                </property>
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_6">
                 <item>
                  <widget class="QLabel" name="label_4">
                   <property name="styleSheet">
                    <string notr="true">*{
	color: white;
	font-family: Noto Sans;
	font-size: 16px;
}</string>
                   </property>
                   <property name="text">
                    <string>Saisissez le nombre total d'avocats avec PLU 4225 vendus</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="plu_4226">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>35</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">*{
	color: rgb(197, 197, 197);
	background: rgba(0, 0, 0, 0.0);
	border: 2px solid;
	border-radius: 1px;
	border-bottom-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:1, stop:0 rgba(255, 121, 215, 255), stop:1 rgba(145, 94, 255, 255));
	border-top-color: rgba(0, 0, 0, 0.0);
	border-right-color: rgba(0, 0, 0, 0.0);
	border-left-color: rgba(0, 0, 0, 0.0);
}</string>
                   </property>
                   <property name="placeholderText">
                    <string>Nombre total d'avocats avec PLU 4225 vendus</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QFrame" name="frame_8">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>100</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">*{
	background-color: rgb(29, 16, 48);
}</string>
                </property>
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_7">
                 <item>
                  <widget class="QLabel" name="label_5">
                   <property name="styleSheet">
                    <string notr="true">*{
	color: white;
	font-family: Noto Sans;
	font-size: 16px;
}</string>
                   </property>
                   <property name="text">
                    <string>Saisissez le nombre total d'avocats avec PLU 4770 vendus</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="plu_4770">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>35</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">*{
	color: rgb(197, 197, 197);
	background: rgba(0, 0, 0, 0.0);
	border: 2px solid;
	border-radius: 1px;
	border-bottom-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:1, stop:0 rgba(255, 121, 215, 255), stop:1 rgba(145, 94, 255, 255));
	border-top-color: rgba(0, 0, 0, 0.0);
	border-right-color: rgba(0, 0, 0, 0.0);
	border-left-color: rgba(0, 0, 0, 0.0);
}</string>
                   </property>
                   <property name="placeholderText">
                    <string>Nombre total d'avocats avec PLU 4770 vendus</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QFrame" name="frame_9">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>100</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">*{
	background-color: rgb(29, 16, 48);
}</string>
                </property>
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_8">
                 <item>
                  <widget class="QLabel" name="label_6">
                   <property name="styleSheet">
                    <string notr="true">*{
	color: white;
	font-family: Noto Sans;
	font-size: 16px;
}</string>
                   </property>
                   <property name="text">
                    <string>Saisissez le nombre total de sac de vendu</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="total_bag">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>35</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">*{
	color: rgb(197, 197, 197);
	background: rgba(0, 0, 0, 0.0);
	border: 2px solid;
	border-radius: 1px;
	border-bottom-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:1, stop:0 rgba(255, 121, 215, 255), stop:1 rgba(145, 94, 255, 255));
	border-top-color: rgba(0, 0, 0, 0.0);
	border-right-color: rgba(0, 0, 0, 0.0);
	border-left-color: rgba(0, 0, 0, 0.0);
}</string>
                   </property>
                   <property name="placeholderText">
                    <string>Nombre total de sac</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QFrame" name="frame_11">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>100</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">*{
	background-color: rgb(29, 16, 48);
}</string>
                </property>
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_9">
                 <item>
                  <widget class="QLabel" name="label_7">
                   <property name="styleSheet">
                    <string notr="true">*{
	color: white;
	font-family: Noto Sans;
	font-size: 16px;
}</string>
                   </property>
                   <property name="text">
                    <string>Saisissez le nombre de petit sac vendu</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="small_bag">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>35</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">*{
	color: rgb(197, 197, 197);
	background: rgba(0, 0, 0, 0.0);
	border: 2px solid;
	border-radius: 1px;
	border-bottom-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:1, stop:0 rgba(255, 121, 215, 255), stop:1 rgba(145, 94, 255, 255));
	border-top-color: rgba(0, 0, 0, 0.0);
	border-right-color: rgba(0, 0, 0, 0.0);
	border-left-color: rgba(0, 0, 0, 0.0);
}</string>
                   </property>
                   <property name="placeholderText">
                    <string>Nombre de petits sac vendu</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QFrame" name="frame_12">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>100</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">*{
	background-color: rgb(29, 16, 48);
}</string>
                </property>
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_10">
                 <item>
                  <widget class="QLabel" name="label_8">
                   <property name="styleSheet">
                    <string notr="true">*{
	color: white;
	font-family: Noto Sans;
	font-size: 16px;
}</string>
                   </property>
                   <property name="text">
                    <string>Saisissez le nombre de gros sac vendu</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="large_bag">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>35</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">*{
	color: rgb(197, 197, 197);
	background: rgba(0, 0, 0, 0.0);
	border: 2px solid;
	border-radius: 1px;
	border-bottom-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:1, stop:0 rgba(255, 121, 215, 255), stop:1 rgba(145, 94, 255, 255));
	border-top-color: rgba(0, 0, 0, 0.0);
	border-right-color: rgba(0, 0, 0, 0.0);
	border-left-color: rgba(0, 0, 0, 0.0);
}</string>
                   </property>
                   <property name="placeholderText">
                    <string>Nombre de gros sac vendu</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QFrame" name="frame_13">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>100</height>
                 </size>
                </property>
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <item>
                  <widget class="QPushButton" name="validButton">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>45</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>500</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">*{
	color: white;
	border-radius: 22px;
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:1, stop:0 rgba(255, 121, 215, 255), stop:1 rgba(145, 94, 255, 255));
}</string>
                   </property>
                   <property name="text">
                    <string>Valider</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
